bim2sim/plugins/PluginTEASER/bim2sim_teaser/utilities/visualize_spaces.py

```diff
@@ -0,0 +1,135 @@
+"""
```

Comment:
please move/keep this file to/in the original bim2sim/utilities/visualize_spaces.py, so we can add more plot functions afterwards from other Plugins (e.g., PluginEnergyPlus utils)